feat: new introductory javascript project (#54305)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-pyramid-generator/660eebd83100d37862268781.md

---
id: 660eebd83100d37862268781
title: Step 2
challengeType: 0
dashedName: step-2
---

# --description--

Your `character` variable currently does not have a value. You can assign a value using the <dfn>assignment</dfn> operator `=`. For example:

```js
let hello = "Hello";
```

> When you assign a value at the same time as declaring a variable, it is called <dfn>initialization</dfn>.

Reviewer: ```suggestion
Assigning a value to a variable at the moment of its declaration is known as <dfn>initialization</dfn>.
```

Reviewer: This makes it a little more clear, I think.

> Initialize your `character` variable by assigning the value `"Hello"` in the declaration.

Reviewer: ```suggestion
Initialize your `character` variable by assigning it the value `"Hello"` during its declaration.
```
